options/locale/locale_en-US.ini

```diff
@@ -1814,6 +1814,16 @@ settings.choose_branch = Choose a branch…
 settings.no_protected_branch = There are no protected branches.
 settings.edit_protected_branch = Edit
 settings.protected_branch_required_approvals_min = Required approvals cannot be negative.
+settings.tags = Tags
+settings.tags.protection = Tag Protection
+settings.tags.protection.pattern = Tag Pattern
+settings.tags.protection.allowed = Allowed
+settings.tags.protection.allowed.users = Allowed users
+settings.tags.protection.allowed.teams = Allowed teams
+settings.tags.protection.allowed.noone = No One
+settings.tags.protection.create = Protect Tag
+settings.tags.protection.none = There are no protected tags.
+settings.tags.protection.pattern.description = Wildcards such as <code>v*</code> or <code>*-release</code> are supported. See <a href="https://godoc.org/github.com/gobwas/glob#Compile">github.com/gobwas/glob</a> documentation for pattern syntax.
```

Comment:
Maybe `https://pkg.go.dev/github.com/gobwas/glob#Compile` is a better link.

Reply:
I changed the other two occurrences too.